user_tools/tests/spark_rapids_tools_e2e/resources/scripts/setup_env.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+err () {
+    echo "ERROR: $1" >&2
+    exit 1
+}
+
+if [ -z "$E2E_TEST_TOOLS_DIR" ]; then
+  err "Please set E2E_TEST_TOOLS_DIR to the root directory of the spark-rapids-tools repository. Exiting script."
+fi
+
+if [ -z "$E2E_TEST_SPARK_BUILD_VERSION" ]; then
+  err "Please set E2E_TEST_SPARK_BUILD_VERSION to the version of Spark used for building Tools JAR. Exiting script."
+fi
+
+if [ -z "$E2E_TEST_HADOOP_VERSION" ]; then
+  err "Please set E2E_TEST_HADOOP_VERSION to the version of Hadoop used for building Tools JAR. Exiting script."
+fi
+
+build_jar() {
+  local jar_tools_dir="$E2E_TEST_TOOLS_DIR/core"
+  echo "Building Spark RAPIDS Tools JAR file"
+  pushd "$jar_tools_dir"
+  mvn install -DskipTests -Dbuildver="$E2E_TEST_SPARK_BUILD_VERSION" -Dhadoop.version="$E2E_TEST_HADOOP_VERSION"
+  popd
+}
+
+install_python_package() {
+  if [ -z "$E2E_TEST_VENV_DIR" ]; then
+    err "Please set E2E_TEST_VENV_DIR to the name of the virtual environment. Exiting script."
+  fi
+
+  echo "Setting up Python environment in $E2E_TEST_VENV_DIR"
+  local python_tools_dir="$E2E_TEST_TOOLS_DIR/user_tools"
+  python -m venv "$E2E_TEST_VENV_DIR"
```

Comment:
can we use a different name then just venv to help with name conflicting?  maybe just venv_e2e_tests or something

Reply:
Modified the venv name to `spark_rapids_tools_e2e_tests_venv`

Reply:
I don't see the change?

Reply:
Virtual Env name is set in the tox.ini file as follows:

https://github.com/NVIDIA/spark-rapids-tools/blob/f271cb30e49d6629f65645b21f48359dbbc4d062/user_tools/tox.ini#L84